integrations/google-analytics/lib/index.js

```diff
@@ -580,6 +580,25 @@ function formatValue(value) {
   return Math.round(value);
 }
 
+function getProductPosition(item, products) {
+  var position = item.properties().position;
+  if (
+    typeof position !== 'undefined' &&
+    !Number.isNaN(Number(position)) &&
+    Number(position) > -1
```

Comment:
Should this also check `Number.isInteger` ?

Reply:
Good point, I'll make that extra check befire the release

Reply:
Sadly `Number.isInteger` is not compatible with early versions of Edge :S I'll leave to google to decide what to do with a floating point number.